components/shooter.py

```diff
@@ -0,0 +1,32 @@
+from magicbot import tunable
+import wpilib
+
+
+class Shooter:
+    shooter_motor: wpilib.Talon
+    shoot_speed = tunable(1.0)  # speed is tunable via NetworkTables
+
+    def __init__(self):
+        self.deployed = False
+
+    def shoot(self):
+        pass
+
+    def stop(self):
+        pass
+
+    def deploy(self):
```

Comment:
What's the difference between the method of shoot and deploy

Reply:
change the function identifyer to shoot?